index.html

```diff
@@ -759,15 +759,15 @@ <h2>Identifier</h2>
     </section>
 
     <section id="http-baseline-profile-data-model">
```

Comment:
section id can be adjusted as well

Reply:
There's significant cleanup in #203 which fixes this and other name problems (Core vs. baseline). 
Suggest to merge this PR before we merge #203.